src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+//new class Bulldozer
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
```

Comment:
The message 'Bulldozer started its work' could be more informative by specifying that it is a Bulldozer that has started working. For example: 'Bulldozer is now working.'